pkg/store/bucket_test.go

```diff
@@ -1216,7 +1231,7 @@ func benchBucketSeries(t testutil.TB, samplesPerSeries, totalSeries int, request
 			Series:           seriesPerBlock,
 			PrependLabels:    extLset,
 			Random:           random,
-			SkipChunks:       t.IsBenchmark(),
+			SkipChunks:       false,
```

Comment:
I think it makes more sense to not skip chunks when creating buckets because that's the real scenario.

Reply:
I think this was fine, but just confusing. So this controls expected `bSeries` if they have chunks or not. Series itself in bucket have chunks in any case. Maybe we can rename this field to be `SkipChunkInExpectedSeries` also it's fine to keep false all the time, just it will use more mem to run benchmark.